scripts/install.sh

```
@@ -0,0 +1,244 @@
+  enable_corepack_step
+  ensure_gh
+  install_patternfly_cli
+
+  printf '\n'
+  printf 'SUCCESS: PatternFly CLI is installed.\n'
+  printf '\n'
+  printf 'Run the CLI with:\n'
+  printf '  patternfly-cli --help\n'
+  printf '\n'
+  printf 'If you just installed nvm, open a new terminal or run:\n'
+  printf '  export NVM_DIR="$HOME/.nvm" && [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"\n'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)